test/array_parameters.toml

```diff
@@ -0,0 +1,14 @@
+[array_parameter_1]
```

Comment:
Can we move all the toml files into a new folder, `test/toml/`?

Reply:
Done